app/jobs/socure_docv_results_job.rb

```diff
@@ -12,10 +12,10 @@ def perform(document_capture_session_uuid:)
     dcs = DocumentCaptureSession.find_by(uuid: document_capture_session_uuid)
     raise "DocumentCaptureSession not found: #{document_capture_session_uuid}" if !dcs
 
-    # analytics = create_analytics(
-    #   user: dcs.user,
-    #   service_provider_issuer: dcs.issuer,
-    # )
+    @analytics = create_analytics(
+      user: dcs.user,
+      service_provider_issuer: dcs.issuer,
+    )
```

Comment:
since docv_result_request.rb already has document_capture_session we do not need to make any changes in socure_docv_results_job.rb.  this can all be done in adocv_result_request.rb